### PR TITLE
Rpy2 (REVIEW)

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: test this please

File: easybuild/easyconfigs/r/rpy2/rpy2-2.4.4-goolf-1.4.10-Python-2.7.5.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+source_urls = ['https://pypi.python.org/packages/source/r/rpy2/']
+sources = [SOURCE_TAR_GZ]
+
+python = 'Python'
+pyver = '2.7.5'
+versionsuffix = '-%s-%s' % (python, pyver)
+
+dependencies = [
+    (python, pyver),
+    ('R', '3.1.0')
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/r/rpy2/rpy2-2.4.4-goolf-1.4.10-Python-2.7.5.eb]: i recommend updates to most recent Python/R, but otherwise this looks good.

Comment (boegel) [inline easybuild/easyconfigs/r/rpy2/rpy2-2.4.4-goolf-1.4.10-Python-2.7.5.eb]: Jenkins says no on current state:

```
R/3.1.0-goolf-1.4.10
```

We do have `3.0.2` with that toolchain. And we have `3.1.2` easyconfigs that come close (goolf*)

Comment (fgeorgatos) [inline easybuild/easyconfigs/r/rpy2/rpy2-2.4.4-goolf-1.4.10-Python-2.7.5.eb]: one more push for this please

lgtm , put this in REVIEW

**Comment by migueldiascosta** [comment]
@pescobar, this PR is being closed for the following reason(s): no activity for > 6 months.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**